### PR TITLE
[WIP] 6959 fix redirects

### PR DESCRIPTION
**Pull request by Tr4pSt3R**
[Review on Reviewable](https://reviewable.io/reviews/moneyadviceservice/cms/299)

### CODE REVIEW

File: app/models/redirect_version.rb [hunk 1]
```diff
@@ -4,4 +4,12 @@ class RedirectVersion < PaperTrail::Version
   def user
     Comfy::Cms::User.find(whodunnit)
   end
+
+  def updated_by
+    begin
+      user.email_local_part
+    rescue ActiveRecord::RecordNotFound => e
+      "deleted user"
```
Comment (tomas-stefano) [inline app/models/redirect_version.rb]: I would use a locale and singue quotes.

**Comment by Tr4pSt3R** [comment]
The redirect system is not available in Welsh. This is the only place where it is being used.

**Comment by tomas-stefano** [comment]
Is this ok to be reviewed?

**Comment by Tr4pSt3R** [comment]
@tomas-stefano this is on hold the moment because of the interruption by the work on the Contento-Azure migration. I will stick a WIP on it

**Comment by tomas-stefano** [comment]
Ok. Thanks!

**Comment by Tr4pSt3R** [comment]
Look here for a cleaner PR: https://github.com/moneyadviceservice/cms/pull/316

**Pull request closed without merging**